subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/service/TestRunParams.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.instrumentation.service
+
+import com.avito.android.stats.SeriesName
+
+/**
+ * Parameters for a specific single test suite run
+ * Don't put global test runner parameters here
+ * "global" - means that it is available on a build start
```

Comment:
Where is this "global"? I don't see it in the class

Reply:
Global is parameters of service itself